OpenTabletDriver.Desktop/Binding/BindingHandler.cs

```diff
@@ -72,6 +93,31 @@ private void HandleMouseReport(TabletReference tablet, IMouseReport report)
             MouseScrollUp?.Invoke(tablet, report, report.Scroll.Y > 0);
         }
 
+        private void HandleWheelButtonReport(TabletReference tablet, IWheelButtonReport report)
+        {
+            HandleBindingCollection(tablet, report, WheelButtons, report.WheelButtons);
+        }
+
+        private void HandleAbsoluteWheelReport(TabletReference tablet, IAbsoluteWheelReport report)
+        {
+            int? delta = (int?)((((int?)report.Position - lastWheelPosition + threeHalfWheelSteps) % wheelSteps) - halfWheelSteps);
+            HandleRelativeWheelReport(tablet, report, delta);
+            lastWheelPosition = report.Position;
+        }
+
+        private void HandleRelativeWheelReport(TabletReference tablet, IDeviceReport report, int? delta)
+        {
+            currentWheelDelta += delta ?? 0;
+
+            ClockwiseRotation?.Invoke(tablet, report, currentWheelDelta);
+            CounterClockwiseRotation?.Invoke(tablet, report, -currentWheelDelta);
```

Comment:
this should probably only emit if the delta is > 0 for their respective state:
```suggestion
            if (currentWheelDelta > 0)
                ClockwiseRotation?.Invoke(tablet, report, currentWheelDelta);
            if (currentWheelDelta < 0)
                CounterClockwiseRotation?.Invoke(tablet, report, -currentWheelDelta);
```
Unless I'm misunderstanding the implementation?

Reply:
Me when the volume goes 📉 permanently when pishing the volume down binding